gapic/templates/examples/sample.py.j2

```diff
@@ -0,0 +1,50 @@
+{#
+ # Copyright (C) 2019  Google LLC
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+#}
+{# Input parameters: sample #}
+{#                   fileHeader#}
+{#                   imports #}
+{#                   callingForm #}
+{#                   callingFormEnum #}
+{# Note: this sample template is WILDLY INACCURATE AND INCOMPLETE #}
+{# It does not correctly enums, unions, top level attributes, or various other things #}
+{% import "feature_fragments.j2" as frags %}
+{{ frags.sample_header(fileHeader, sample, callingForm) }}
+
+# [START {{ sample.id }}]
+{# python code is responsible for all transformations: all we do here is render #}
+{% for importStatement in imports %}
+{{ importStatement }}
+{% endfor %}
+
+{# also need calling form #}
+def sample_{{ frags.renderMethodName(sample.rpc) }}({{ frags.printInputParams(sample.request) }}):
+    """{{ sample.description }}"""
+
+    # [START {{ sample.id }}_core]
```

Comment:
We no longer include the `_core` region tag pair for now (cf https://github.com/googleapis/gapic-generator/pull/2810)

We may add it back when we update our plan for dealing with reference page (APIs Explorer) samples.

Reply:
Good to know. The region tag comments are one of the (thankfully) few remaining cargo cult elements of the rendered output.